go.mod

```diff
@@ -3,20 +3,17 @@ module github.com/jaypipes/ghw
 go 1.21
 
 require (
-	github.com/StackExchange/wmi v1.2.1
-	github.com/jaypipes/pcidb v1.0.1
+	github.com/jaypipes/pcidb v1.1.0
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v1.8.0
+	github.com/spf13/cobra v1.9.1
+	github.com/yusufpapurcu/wmi v1.2.4
```

Comment:
is this better? we moved to a single-maintainer dependency with a PR open since april with no comments:
https://github.com/yusufpapurcu/wmi/pull/10
last commit and release 1.5 years ago